app/main/views/g_cloud.py

```diff
@@ -365,7 +365,7 @@ def save_search(framework_framework):
             form.name.errors = ["Names must be between 1 and 100 characters"]
 
         if not save_search_selection and request.method == "POST":
-            save_search_selection_error = "Please choose where to save your search."
+            save_search_selection_error = "Please choose where to save your search"
```

Comment:
Can we refactor this so that the content is in the template? Ideally as much content as possible should be there rather than in code files.

Reply:
yeah we can do

Reply:
I have reviewed this and in another PR, this error message has been used in multiple places inside the template. Therefore I think this is still the best place to have the content.

Reply:
https://projects.invisionapp.com/share/QUDX63O3K#/screens/258002852

Reply:
Hmm, okay, that's probably sensible enough then. 👍